tutorials/plot_09-Reporting.py

```diff
@@ -0,0 +1,169 @@
+"""
+09: Reporting & Referencing
+===========================
+
+This section covers how to access reporting info and reference use of the module.
+
+This page is a hands-on example of the reporting and referencing information on the
+`Reference page <https://fooof-tools.github.io/fooof/reference.html>`_.
+"""
+
+###################################################################################################
+
+# Import FOOOF model objects
+from fooof import FOOOF, FOOOFGroup
+
+# Import simulation functions to create some example data
+from fooof.sim import gen_power_spectrum, gen_group_power_spectra
+
+# Import utilities to print out information for reporting
+from fooof.utils.reports import methods_report_info, methods_report_text
+
+# sphinx_gallery_start_ignore
+# Note: this code gets hidden, but serves to create the text plot for the icon
+from fooof.core.strings import gen_methods_report_str
+from fooof.core.reports import REPORT_FONT
+import matplotlib.pyplot as plt
+text = gen_methods_report_str(concise=True)
+text = text[0:142] + '\n' + text[142:]
+_, ax = plt.subplots(figsize=(8, 3))
+ax.text(0.5, 0.5, text, REPORT_FONT, ha='center', va='center')
+ax.set_frame_on(False)
+_ = ax.set(xticks=[], yticks=[])
+# sphinx_gallery_end_ignore
+
+###################################################################################################
+# Checking Module Version
+# -----------------------
+#
+# It's useful and important to keep track of which version of the module you are using,
+# and to report this information when referencing use of the tool.
+#
+# There are several ways to check the version of the the module that you are using,
+# including checking what is installed in the Python environment you are using.
+#
+# From within Python, you can also check the version of the module by checking
+# `__version__`, as shown below:
+#
+
+###################################################################################################
+
+# Check the version of the module
+from fooof import __version__ as fooof_version
+print('Current fooof version:', fooof_version)
+
+###################################################################################################
+# Getting Model Reporting Information
+# -----------------------------------
+#
+# To assist with reporting information, the module has the following utilities:
+# - the :func:`~.methods_report_info`, which prints out methods reporting information
+# - the :func:`~.methods_report_text`, which prints out a template of methods reporting text
```

Comment:
the dashes don't get built as bullet points. not sure if that was intended or not

Reply:
I always forget the blank line before a bulleted list! Fixed!